test/run_tests.sh

```diff
@@ -14,7 +14,7 @@ run_jm_tests ()
     export C_INCLUDE_PATH="${C_INCLUDE_PATH}:${VIRTUAL_ENV}/include"
 
     pushd "${jm_source}"
-    curl --retry 5 -L https://github.com/JoinMarket-Org/miniircd/archive/master.tar.gz -o miniircd.tar.gz
+    curl --retry 5 -L https://github.com/jrosdahl/miniircd/archive/432513e702793f5b0530f6865c0c7c4f83ecccc6.tar.gz -o miniircd.tar.gz
```

Comment:
why is this needed?

Reply:
It's been rebased now (the Joinmarket-Org version)